lib/portlayer/storage/config.go

```diff
@@ -31,4 +31,6 @@ type Configuration struct {
 	// Keyed by the volume store name (which is used by the docker user to
 	// refer to the datstore + path), valued by the datastores and the path.
 	VolumeLocations map[string]url.URL `vic:"0.1" scope:"read-only"`
+
+	ScratchSize int `vic:"0.1" scope:"read-only" key:"scratch_size"`
```

Comment:
Yeah Emma caught this here https://github.com/vmware/vic/pull/1982/files#r74835090 <- I'll alter the flag to provide a default there so that 8G will be provided if nothing else is.